suro-kafka-producer/src/test/java/org/apache/kafka/common/metrics/KafkaMetricBuilder.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.kafka.common.metrics;
+
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.utils.Time;
+
+public class KafkaMetricBuilder {
+	
+	public static KafkaMetric newKafkaMetric(Object lock, MetricName metricName, Measurable measurable, MetricConfig config, Time time)
```

Comment:
This is a hack to get around the fact the KafkaMetric only having a package leveled constructor.